src/content-helper/editor-sidebar/smart-linking/provider.ts

```diff
@@ -41,6 +41,23 @@ interface SmartLinkingApiResponse {
  * @since 3.14.0
  */
 export class SmartLinkingProvider {
+	/**
+	 * The AbortController instance used to cancel the fetch request.
+	 *
+	 * @since 3.15.0
+	 */
+	private static abortController: AbortController = new AbortController();
+
+	/**
+	 * Cancels the fetch request.
+	 *
+	 * @since 3.15.0
+	 */
+	static cancelRequest(): void {
+		SmartLinkingProvider.abortController.abort();
+		SmartLinkingProvider.abortController = new AbortController();
```

Comment:
Question for my own understanding: Why do we need to re-instantiate the `abortController` here?

Reply:
The AbortController class is single-use only. Once the `abort` method is called, it cannot be reused again, hence the need of re-instantiate it after the request is cancelled.